src/Employer/Employer.Web/Views/Error/Error.cshtml

```diff
@@ -5,6 +5,6 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">There's been an error</h1>
-        <p>Sorry, there seems to be an error with this page. Please refresh your browser or <a href="/">go back to the homepage</a>.</p>
+        <p>Sorry, there seems to be an error with this page. Please refresh your browser or <a asp-route="@RouteNames.Dashboard_Account_Home">return to homepage</a>.</p>
```

Comment:
Should the return be Return with capital R?

Reply:
no because it is in the middle of a sentence.